### DISCUSSION TITLE
Use Infinispan testcontainer based on jakarta

### DISCUSSION THREAD

**Discussion by davidmestr** [infinispan/infinispan#10771] [General]
We would like to use an org.infinispan:infinispan-server-testdriver-core to do integration test with Spring Boot 3, but this dependency need javax.transaction.

Is there an infinispan tescontainer based on jakarta?

**Comment by tristantarrant** [comment]
I've created https://issues.redhat.com/browse/ISPN-14736

**Reply by alc2020** [reply]
Hello, is there any update on this. I am having issues making infinispan work work with spring boot 3.4.2

**Reply by tristantarrant** [reply]
Infinispan 15.0 is Jakarta-only
